docs/dev/generator.js

```diff
@@ -1002,6 +1003,23 @@ const generateSearch = async () => {
   await fs.writeFile(`${rootDir}/searchData.json`, JSON.stringify(searchData,null,2));
 };
 
+const discussionsData = [];
+discussionsArray.forEach(({ node }) => {
+  const { url, title } = node;
+  discussionsData.push({
+    objectID: url,
+    pt: title,
+    t: sd_p,
+    c: title,
+  });
+});
```

Comment:
I think the query should be run in `/docs/search/index.mjs`, not here

Reply:
[786581b](https://github.com/reach-sh/reach-lang/pull/1113/commits/786581b9769b35184cccfaf16530fbcfb20b19dd)